imports/plugins/included/email-smtp/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,37 @@
+extend type Mutation {
+  "Use this mutation to verify the SMTP email settings"
+  verifySMTPEmailSettings(
+    "Mutation input"
+    input: VerifySMTPEmailSettingsInput!
+    ): VerifySMTPEmailSettingsInputPayload!
+}
+
+"Input for an `VerifySMTPEmailSettingsInput`"
+input VerifySMTPEmailSettingsInput {
+  "The host of the SMTP email service"
+  host: String
+
+  "The password of the SMTP email service user"
+  password: String!
+
+  "The port to send email via"
+  port: Int
+
+  "The SMTP email service"
+  service: String!
+
+  "The ID of the shop this setting belongs to"
+  shopId: ID!
+
+  "The username of the SMTP email service user"
+  user: String!
+}
+
+"Response payload for the verifySMTPEmailSettings mutation"
+type VerifySMTPEmailSettingsInputPayload {
+  "The same string you sent with the mutation params, for matching mutation calls with their responses"
+  clientMutationId: String
+
+  "Are the settings OK"
+  verified: Boolean
```

Comment:
Maybe do `isVerified` to follow the rule about boolean naming. It should be safe to do `Boolean!`, too. I don't think it will ever be `null` when there were no errors.